.rubocop-txi.yml

```diff
@@ -28,6 +28,13 @@ Metrics/LineLength:
 Metrics/MethodLength:
   Max: 20
 
+Rails/HttpPositionalArguments:
+  Exclude:
```

Comment:
Maybe, though it would be weird to run into this outside of controller specs, wouldn't it?